css/app-settings.scss

```diff
@@ -66,14 +66,11 @@
 			display: table;
 
 			label {
-				display: table-cell;
```

Comment:
Default for `<label>` is `inline`. `.multiselect`, which is a `<div>`, doesn't need it.

Reply:
For the <div> tag, in theory, yes, but you have to put the "display: block" anyway because the ".multiselect" class brings an  "inline-block" display before (heritage). For the "label" tag, the default display is "inline". So we must also provide the "block" display.

Reply:
> Default for `<label>` is `inline`. `.multiselect`, which is a `<div>`, doesn't need it.

For the "div" tag, in theory, yes, but you have to put the "display: block" anyway because the ".multiselect" class brings an "inline-block" display before (heritage).